tests/upgrades/test_syncplan.py

```diff
@@ -96,7 +96,6 @@ def test_post_sync_plan_migration(self):
         self.assertEqual(product.sync_plan.id, sync_plan.id)
         self.assertEqual(sync_plan.name, entity_data.get("sync_plan_name"))
         self.assertEqual(sync_plan.interval, entity_data.get("interval"))
-        self.assertEqual(sync_plan.sync_date, entity_data.get("sync_date"))
```

Comment:
I understand this not the end purpose of this test however we should keep this inorder to keep a check after upgrade and it shall add a plus for us too.

Reply:
If some other function or test ran before this test and capsule sync happened then this is likely to fail and unnecessarily causing this fail test. So thought to remove it. I am still at the side of removing it.

Reply:
I did not get this one "capsule sync happened then this is likely to fail" ? How this is going to impact the sync plan 'sync date' ?

Reply:
@vijay8451 I believe that field references when the sync plan will trigger next. So if the sync plan hits between when entity_data is collected and when sync_plan is queried, then we could get a false negative result.
@jyejare feel free to correct me if I'm wrong.

Reply:
@JacobCallahan by default sync plan `Start Date` and `Next Sync` are part of future date so should not get impacted by any upgrade interval.
@jyejare please correct me if i missed any other part in it.

Reply:
@JacobCallahan , You are absolutely correct.

@vijay8451 , That field represents the last sync time and by 'capsule sync' I mean if somehow another test or just capsule CV publish will change that time and as @JacobCallahan  said this test will fail with false negative result.

Reply:
Just removed my last comment as I was confused between sync plan date and capsule sync date.

Reply:
@vijay8451 , Updated the code with sync plan sync date check. Please review.

Reply:
Thanks @jyejare :+1: